firefox-ios/Client/Frontend/Home/Homepage Rebuild/HomepageViewController.swift

```diff
@@ -100,8 +100,6 @@ final class HomepageViewController: UIViewController,
         homepageState = state
         if homepageState.loadInitialData {
```

Comment:
probably can remove this, but lets leave this for now and remove when the collection is more fleshed